src/iop/liquify.c

```diff
@@ -3158,6 +3160,30 @@ int button_pressed(struct dt_iop_module_t *module,
   return handled;
 }
 
+static void _start_new_shape(dt_iop_module_t *module)
+{
+  dt_iop_liquify_gui_data_t *g = (dt_iop_liquify_gui_data_t *) module->gui_data;
+
+  //  create initial shape at the center
+  float complex pt = 0.0f;
+  float scale = 1.0f;
+  get_point_scale(module, 0.5f * darktable.develop->width, 0.5f * darktable.develop->height, &pt, &scale);
+  float radius = 0.0f, r = 1.0f, phi = 0.0f;
+  get_stamp_params(module, &radius, &r, &phi);
+  //  start a new path
+  g->temp = alloc_move_to(module, pt);
+  g->temp->warp.radius = pt + radius;
+  g->temp->warp.strength = pt + r * cexp(phi * I);
+  g->status |= DT_LIQUIFY_STATUS_PREVIEW;
+  g->status |= DT_LIQUIFY_STATUS_NEW;
+
+  g->just_started = TRUE;
```

Comment:
I support this is the issue I'm experiencing. Place a point, then place a curve with at least 3 points. You'll see the mess on the screen :)

Reply:
I may not be seeing what you are seeing. Would you please post a screenshot and/or exact steps so I can reproduce?

Placing a point and then a curve with >3 nodes works fine here. Either with the normal or ctrl/multiple version.

Reply:
Ok, it is not that easy to reproduce, but take the image used for the integration test.

1. clear the history
2. select liquify
3. use the non continuous mode (the continuous has not been tested)
4. place a single point in the center of the perceptional scale (white circle on the right)
5. place a curve 3 points
6. reset the module with the button
7. repeatgit pull https://github.com/dterrahe/darktable.git init_gui_accel 4
8. repeat 5


![image](https://user-images.githubusercontent.com/467069/97791189-e939ed80-1bcf-11eb-93cd-561656d11746.png)

Reply:
Could this have something to do with crap from old points being left behind in the params when deleting or resetting? Similar to #6685 for curves.

I'm putting on my todo list to have a look at this. Management of the point list isn't as obvious in liquify, so I'll need to analyse and do careful testing. Similar issues probably in retouch and spot.